### PR TITLE
Chat sample header - People button focusable

### PR DESCRIPTION
**Pull request by alcail**
# What
Made People button focusable in Chat sample header

# Why
People button was not focusable so user could not navigate to it and thus turn the participant list panel on/off
bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2584873

# How Tested
ran Chat sample locally

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: samples/Chat/src/app/styles/ChatHeader.styles.ts [hunk 1]
```diff
@@ -30,21 +30,21 @@ export const leaveIcon: IIconProps = {
   iconName: 'Leave'
 };
 
-export const panelButtonStyle = mergeStyles({
-  padding: '0px 13px',
-  cursor: 'pointer'
```
Comment (prprabhu-ms) [inline samples/Chat/src/app/styles/ChatHeader.styles.ts:35]: This CSS change seems unrelated? (cursor styling got dropped?)

**Review by prprabhu-ms — APPROVED** [review]

File: samples/Chat/src/app/styles/ChatHeader.styles.ts (hunk 1 above)
Comment (alcail) [reply samples/Chat/src/app/styles/ChatHeader.styles.ts:35]: it is related. No need to force the cursor to pointer as it's done automatically now that we use IconButton component instead of just Icon

**Review by mgamis-msft — APPROVED** [review]
